py_zipkin/logging_helper.py

```diff
@@ -250,9 +279,9 @@ def emit(self, record):
                 " to be provided for {0} span".format(span_name)
             )
         service_name = record.msg.get('service_name', None)
-        # Presence of service_name means new client span.
+        # Presence of service_name means this is to be a new local span.
         if service_name is not None:
-            self.store_client_span(
+            self.store_local_span(
```

Comment:
ps a local span wouldn't have sr, ss, cr, cs. There's only one special case, which is that if there are no annotations or binary annotations, we create a fake one (so that indexing works by service name) called "lc"(local component)

ex. in java, we track to see if anything was created prior, and if nothing, we stamp like this..

Ex when there's no annotation, we'd have never added the local endpoint to the span, so do it as a last-ditch attempt
```
    if ((flags & FLAG_LOCAL_ENDPOINT) == 0) { // create a small dummy annotation
      span.addBinaryAnnotation(BinaryAnnotation.create(LOCAL_COMPONENT, "", localEndpoint));
    }
```